wic/iot2050-qemu.wks.in

```diff
@@ -0,0 +1,20 @@
+#
+# Copyright (c) Siemens AG, 2023
+#
+# Authors:
+#  Enes Çolpan <enes.colpan@siemens.com>
+#
+# This file is subject to the terms and conditions of the MIT License.  See
+# COPYING.MIT file in the top-level directory.
+#
+
+include ebg-sysparts.inc
+
+part --source rawcopy --sourceparams "file=${IMAGE_FULLNAME}.squashfs" --align 1024 --fixed-size 4G --uuid "${ABROOTFS_PART_UUID_A}"
+part --source empty --align 1024 --fixed-size 4G --uuid "${ABROOTFS_PART_UUID_B}"
+
+# persistent partitions
+part /var --source rootfs --rootfs-dir=${IMAGE_ROOTFS}/var --label var --fstype=ext4 --mkfs-extraopts "-T default" --ondisk sda --align 1024  --size 512M
+part /home --source rootfs --rootfs-dir=${IMAGE_ROOTFS}/home --label home --fstype=ext4 --mkfs-extraopts "-T default" --ondisk sda --align 1024  --size 512M
+
+bootloader --ptable gpt --append "rootwait ${EXTRA_KERNEL_PARAMS}"
```

Comment:
Again, too much duplication. Why do we need an own image layout?

Reply:
Removed file and used swu.wks file with a variable.